microcloud/test/includes/microcloud.sh

```diff
@@ -113,7 +123,15 @@ set_debug_binaries() {
 
   if [ -n "${LXD_DEBUG_PATH}" ]; then
     echo "==> Add a debug binary for LXD."
-    lxc exec "${name}" -- rm -f /var/snap/lxd/common/lxd.debug
+
+    # lxc exec may lock up on fast-running commands if running in the background, so we need to sleep 1s.
+    lxc exec "${name}" -- sh -c "
+      sleep 1
+      if test -e /var/snap/lxd/common/lxd.debug ; then
+        rm -f /var/snap/lxd/common/lxd.debug
+      fi
+    "
```

Comment:
Why not move `sleep 1` out and then just `lxc exec "${name}" -- rm -f /var/snap/lxd/common/lxd.debug` which will be a noop if the file isn't there.